package.json

```diff
@@ -24,6 +24,12 @@
     "type": "git",
     "url": "git://github.com/js-cookie/js-cookie.git"
   },
+  "files": [
+    "src/**/*.js",
+    "MIT-LICENSE.txt",
+    "SERVER_SIDE.md",
+    "CONTRIBUTING.md"
```

Comment:
CHANGELOG and CONTRIBUTING are different, though. I don't think npm does anything special with CONTRIBUTING.